File: solutions/1758. Minimum Changes To Make Alternating Binary String/Solution.py
class Solution:
    def minOperations(self, st: str) -> int:
        s = list(st)
        n = len(s)
        count = 0
        for i in range(n - 1):
            if s[i] == s[i+1]:
                s[i+1] = '1' if s[i+1] == '0' else '0'
                count+=1
        return min(count, n - count)

File: solutions/1758. Minimum Changes To Make Alternating Binary String/Solution.ts
function minOperations(s: string): number {
  const n = s.length;
  const sArr: string[] = s.split("");
  let count: number = 0;
  for (let i = 0; i < n; i++) {
    if (sArr[i] === sArr[i + 1]) {
      // sArr[i + 1] = sArr[i + 1] === "0" ? "1" : "0";
      sArr[i + 1] = (1 - +sArr[i + 1]).toString();
      count++;
    }
  }
  return Math.min(count, n - count);
}
